companies/intel.md

```diff
@@ -0,0 +1,22 @@
+---
+fortune_rank: 45
+name: Intel
+tags: ["fortune500", "tech", "computers", "artificial intelligence"]
+image: name-in-images-folder.png
```

Comment:
This should reference an image file.  Would you be able to save the image from their tweet and put it into our images folder?  A name like `intel.png` would be perfect. Then you can reference it here.

Reply:
I am logging an issue to myself to make the instructions for image more clear :)

Reply:
I actually just forgot about the image :sweat_smile:
Now I added the tweets image as requested.